mlir/lib/Target/LLVM/NVVM/Target.cpp

```diff
@@ -167,6 +167,15 @@ class NVPTXSerializer : public SerializeGPUModuleBase {
 
   gpu::GPUModuleOp getOperation();
 
+  std::optional<SmallVector<char, 0>>
```

Comment:
I see that you are only moving this, but consider adding some documentation if you understand what this does.